README.md

```diff
@@ -88,6 +89,29 @@ You can assign attributes for your appliances using three methods:
 
 Once the authentication is prepared and the manifests containing the resources are written, you can use ``` puppet apply <manifest>``` to run your manifests and execute the desired changes to the system.
 
+### Synergy Image Streamer Authentication
+
+The attributes required for authenticating with your HPE Synergy Image Streamer Appliance are:
+
+* `I3S_URL` - The web address for the HPE Image Streamer appliance. For example, https://imagestreamer.example.com
+* `I3S_TOKEN` - Session token used for authentication.
+* `I3S_API_VERSION` - This defaults to the 300 API version (Minimum supported for Synergy hardware variant).
+* `I3S_LOG_LEVEL` - The log level of the HPE ImageStreamer appliance. This defaults to **info**
+* `I3S_SSL_ENABLED` - HPE recommends setting this value to **true**
+
+
+You can assign attributes for your appliances using three methods:
+
+- Create a json file named `login_i3s.json` on your working directory, and enter the authentication information for your appliance. See [login_i3s.json](examples/login_i3s.json) for an example.
```

Comment:
Yes, I agree. I will apply this change for the [OneView Appliance Authentication](https://github.com/HewlettPackard/oneview-puppet#appliance-authentication) section as well.